src/subscriptions/restHookLambda/restHook.ts

```diff
@@ -19,15 +21,29 @@ const mergeRequestHeaders = (allowListHeader: string[], channelHeader: string[])
     const mergeHeader = (header: string) => {
         const colonIndex = header.indexOf(':') === -1 ? header.length : header.indexOf(':');
         const headerKey = header.substring(0, colonIndex);
-        const headerValue = header.substring(colonIndex + 1);
-        mergedHeader[headerKey] = headerValue;
+        mergedHeader[headerKey] = header.substring(colonIndex + 1);
     };
 
     allowListHeader.forEach((header) => mergeHeader(header));
     channelHeader.forEach((header) => mergeHeader(header));
     return mergedHeader;
 };
 
+/**
+ * Push latency metric to CloudWatch
+ * @param messages
+ */
+const pushLatencyMetric = metricScope((metrics) => async (messages: SubscriptionNotification[]): Promise<void> => {
```

Comment:
nit: rename to something like `recordLatencyMetric` or ``logLatencyMetric``. I think that "push" sounds misleading